pom.xml

```diff
@@ -59,7 +59,7 @@
         <jacoco.version>0.8.8</jacoco.version>
         <maven-plugin-tools.version>3.6.4</maven-plugin-tools.version>
         <jbool.version>1.24</jbool.version>
-        <mu-logging.version>3.0.3</mu-logging.version>
+        <mu-logging.version>3.0.4</mu-logging.version>
```

Comment:
Why it didn't caught by our dependency bots?

Reply:
Maybe it was.

The reason for this particular change is different: it re-enables building Diktat with Java 8.